packages/user-interface/index.js

```diff
@@ -1,34 +1,48 @@
 import './style.css';
 
-export const showNotification = ({ body, type = 'info', id = 'common' }) => {
+export const showNotification = ({
+  body,
+  clickHandler = () => {},
+  type = 'info',
+  id = 'common',
+}) => {
   const typeClass = {
-    info: 'flash-info',
-    warn: 'flash-warn',
-    error: 'flash-error',
+    info: 'Toast--info',
+    success: 'Toast--success',
+    warn: 'Toast--warning',
+    error: 'Toast--error',
+  };
+
+  const icons = {
+    info:
+      '<svg style="pointer-events: none;" class="octicon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path fill-rule="evenodd" d="M8 1.5a6.5 6.5 0 100 13 6.5 6.5 0 000-13zM0 8a8 8 0 1116 0A8 8 0 010 8zm6.5-.25A.75.75 0 017.25 7h1a.75.75 0 01.75.75v2.75h.25a.75.75 0 010 1.5h-2a.75.75 0 010-1.5h.25v-2h-.25a.75.75 0 01-.75-.75zM8 6a1 1 0 100-2 1 1 0 000 2z"></path></svg>',
```

Comment:
The svgs GitHub uses on the notification page don't have `point-events` set, I don't think that's needed if the click handler is moved to the close button.

GitHub's also setting `aria-hidden="true"` on these.

Reply:
The way how we set the click handler requires us to set this css property, otherwise `event.target` is the `svg`-tag